sdk/cosmos/azure-cosmos-benchmark/pom.xml

```diff
@@ -90,7 +90,7 @@ Licensed under the MIT License.
     <dependency>
       <groupId>io.netty</groupId>
       <artifactId>netty-tcnative</artifactId>
-      <version>2.0.26.Final</version> {x-version-update;io.netty:netty-tcnative;external_dependency}
+      <version>2.0.27.Final</version> {x-version-update;io.netty:netty-tcnative;external_dependency}
```

Comment:
Should this use the Cosmos specific tag instead of the general one? Should we use this PR to get the general and Cosmos versions inline?

Reply:
@alzimmermsft please update cosmos versions to reactor-core - 3.3.3
in external_dependencies.txt file. 
For now, keep using cosmos versions.

Reply:
Looks like this dependency is commented out but in general, since cosmos is preparing for release, it's better to check with the Cosmos team if they are willing to update the dependency version close to the release. But after release, we should consolidate and use the same versions in Cosmos and other track 2 libraries.

Reply:
I agree with @srnagar  Yeah, let's use the same versions, but have diff sources - cosmos will take from cosmos dependency section.. 
We have an issue with netty-tcnative.. We are investigating it ..

Reply:
I'll update the version 😄